docs/customization.rst

```diff
@@ -0,0 +1,42 @@
+Customization
+===============
+
+Besides installing plugins and changing settings,
+several other aspects of SublimeLinter can be customized.
+
+Context Menu
+------------
+
+Removing context menu entries works via ST's standard override protocol.
+
+- In Packages create a directory with the same name as the package you want to override
+  (in this case "SublimeLinter", but this also works for AlignTab or ColorConvert to name a few).
+- Using `PackageResourceViewer <https://packagecontrol.io/packages/PackageResourceViewer>`_
+  open the Context.sublime-menu file from that package,
+  and save it in the new directory you just created.
+- This file will now replace the one from the installed package.
+  By the way, you can override every single file in any package this way.
+- To remove the context menu, change the ``children`` key to an empty array ``[]``.
+
```

Comment:
I just installed PRV.  Looks like I can edit away the SublimeMerge ads.  But also looks like it's way easier (or is it only with the default packages). I just used "PRV: Open Resource", then navigate through the quick panel ~~hell~~ (it's super slick, actually) into `Default > Context` which opens the file with the correct path set, ready to edit and save. Away with the ads.

Reply:
PRV does not recurse into sub directories.  We should move the two menu files out of the sub dir "menus" to the top level.  I never find them anyway?

Reply:
Excellent. IIRC you can browse into sub directories with PRV. Regardless, we should move the menus to the top level, I can never find them either.